platform-bukkit/src/main/java/net/kyori/adventure/platform/bukkit/BukkitAudiencesImpl.java

```diff
@@ -134,6 +133,7 @@ static BukkitAudiences instanceFor(final @NotNull Plugin plugin) {
 
   @Override
   protected @NotNull BukkitAudience createAudience(final @NotNull Collection<CommandSender> viewers) {
+    this.softDepend("ViaVersion");
```

Comment:
This doesn't need to run on every audience creation -- maybe put it in the computeIfAbsent lambda within the build() method of the BukkitAudiences.Builder?